js/main.js

```diff
@@ -0,0 +1,25 @@
+// Первая функция
+
+function getRandomNumber(min, max) {
+  let temp;
+  if (max <= min) {
+    temp = min;
+    min = max;
+    max = temp;
+  }
```

Comment:
Вот здесь можно обойтись без временной переменной и использовать деструктуризацию
```suggestion
  if (max <= min) {
    [min, max] = [max, min];
  }
```